src/Search/table.css

```diff
@@ -0,0 +1,15 @@
+.table {
+    font-family: arial, sans-serif;
+    border-collapse: collapse;
+    width: 50%;
+}
+
+.td, th {
```

Comment:
do not style tag names (like `th`). instead use classnames. this provides better flexibility.